src/config/db_search_words.txt

```diff
@@ -1,18 +1,10 @@
-password|||10|||triggers unwanted classes like password reset, hence the low score
-privatekey|||80
-private_key|||80
-apikey|||75
-http:|||10
-https:|||7
-database_secret|||80
-database_password|||80
-databasepassword|||80
-databasesecret|||80
-(https|http):\/\/.*api.*|||60||| This regex matches any URL containing 'api'
-(https|http):\/\/.*test.*|||60||| This regex matches any URL containing 'test'
-(https|http):\/\/.*uat.*|||60||| This regex matches any URL containing 'uat'
+passw(d|ord)?|||10|||triggers unwanted classes like password reset, hence the low score
```

Comment:
Add support for pwd ? Handle camel case to detect vars that might hold interesting information at some point ? Is passw actually used in applications to justify the `?` ?

`(P|p)(wd|assw(d|ord))`

Maybe the case is handled, didn't check in the code.

Reply:
Thanks for your comment. In a hour I will push a new commit to the development branch with also updated wordlists.

`Maybe the case is handled, didn't check in the code.` -> The code checks all regex's case insensitive.